### DISCUSSION TITLE
Whre is the ultrapeer version?

### DISCUSSION THREAD

**Discussion by JamesOlvertone** [zlatinb/muwire#108] [General]
I read somewhere here that MW ist delievered as an lead version and an ultrapeer version?

Where can I download or how can I build the up version of MW?

**Comment by zlatinb** [comment]
Hi, currently there only exists an Ultrapeer version.  The leaf version is not implemented yet.  What you get by building from source is the Ultrapeer version.

**Comment by JamesOlvertone** [comment]
ok thank you.
